src/instanceMethods/keydown-handler.js

```diff
@@ -133,10 +120,3 @@ const handleArrows = (key) => {
     buttonToFocus.focus()
   }
 }
-
-const handleEsc = (e, innerParams, dismissWith) => {
```

Comment:
revert this along with https://github.com/sweetalert2/sweetalert2/pull/2191/files#r597524197